test/recipes/30-test_evp_data/evpmd_sha.txt

```diff
@@ -58,6 +58,22 @@ Input = "a"
 Ncopy = 64
 Output = cdc76e5c9914fb9281a1c7e284d73e67f1809a48a497200e046d39ccc7112cd0
 
+Digest = SHA256-192
+Input = "abc"
+Output = ba7816bf8f01cfea414140de5dae2223b00361a396177a9c
+
+Digest = SHA256-192
+Input = "abcdbcdecdefdefgefghfghighijhijkijkljklmklmnlmnomnopnopq"
+Output = 248d6a61d20638b8e5c026930c3e6039a33ce45964ff2167
+
+Digest = SHA256-192
+Input = "a"
+Ncopy = 288
+Count = 3472
+Input = "a"
+Ncopy = 64
```

Comment:
Duplication of Ncopy and Input??

Reply:
Test input is built up cumulatively, so the extra line add another 64 `a`s to the end of the test input (for a total of exactly 1,000,000)

The same input is used for SHA256 above, I just copied those tests